openlibrary/core/wikidata.py

```diff
@@ -63,6 +69,97 @@ def to_wikidata_api_json_format(self) -> str:
         }
         return json.dumps(entity_dict)
 
+    def get_description(self, language: str = 'en') -> str | None:
+        """If a description isn't available in the requested language default to English"""
+        return self.descriptions.get(language) or self.descriptions.get('en')
+
+    def get_external_profiles(self, language: str = 'en') -> list[dict]:
+        """
+        Get formatted social profile data for all configured social profiles.
+
+        Returns:
+            List of dicts containing url, icon_url, and label for all social profiles
+        """
+        profiles = []
+        profiles.extend(self._get_wiki_profiles(language))
+
+        for profile_config in SOCIAL_PROFILE_CONFIGS:
+            values = self._get_statement_values(profile_config["wikidata_property"])
+            profiles.extend(
+                [
+                    {
+                        "url": f"{profile_config['base_url']}{value}",
+                        "icon_url": f"/static/images/identifier_icons/{profile_config["icon_name"]}",
```

Comment:
```suggestion
                        "icon_url": f"/static/images/identifier_icons/{profile_config['icon_name']}",
```
?

Reply:
In python 3.12+, this is actually ok! They rewrote the parser to make f-strings more resilient :) https://peps.python.org/pep-0701/

Reply:
Oooh. Neat!